server/middleware/user.js

```diff
@@ -56,13 +55,7 @@ async function addUser(req, res, next) {
         const response = await infoService.post('/user', body, config);
         res.send({ userUUID: response.data.userUUID });
     } catch (error) {
-        if (isAxiosErrorWithCode(error, 409)) { // 409 errors are thrown when the user already exists and are logged in info service
-            logger.warn(error);
-            next(new UserAlreadyExistsError(error.response.data, error.response.status));
-        } else {
-            logger.error(error);
-            next(new FormSubmissionError(error.response.data, error.response.status));
-        }
+        next(new FormSubmissionError(error.response.data, error.response.status));
```

Comment:
Is this just going to display the error from info-service to the user? it looks like the info-service error sends the UUID, which isn't the best error message. Perhaps it should just silently succeed and redirect to the existing user's management page?

Reply:
info-service will throw back the message provided in the Exception, so in this case "User with provided email address already exists"

Line 66 on https://github.com/UKHomeOffice/hocs-info-service/pull/446/files#diff-55c46c9183cdfc580cfdc660eb2f524aeaa6cdeafd0e30573e362226cf1f8480